package.json

```diff
@@ -112,6 +112,7 @@
   },
   "peerDependencies": {
     "ember-source": "^3.28.0 || ^4.0.0",
+    "qunit": "^2.19.4",
```

Comment:
Let's make this a bit more permissive and just do `^2.0.0` here.

Reply:
```suggestion
    "qunit": "^2.0.0",
```

Reply:
sure thing.